src/liboslexec/shadingsys.cpp

```diff
@@ -265,9 +265,26 @@ ShadingSystem::execute (ShadingContext &ctx, ShaderGroup &sas,
 
 
 const void*
-ShadingSystem::get_symbol (ShadingContext &ctx, ustring name, TypeDesc &type)
+ShadingSystem::get_symbol (ShadingContext &ctx, ustring layername,
```

Comment:
Right. If they know what's good for them, they won't ask for symbols that they didn't say would potentially be used as renderer outputs.